security/audit-log-view.adoc

```diff
@@ -33,4 +34,4 @@ ifndef::openshift-rosa,openshift-dedicated[]
 * xref:../security/audit-log-policy-config.adoc#audit-log-policy-config[Configuring the audit log policy]
 endif::[]
 * xref:../observability/logging/log_collection_forwarding/log-forwarding.adoc#log-forwarding[About log forwarding]
-
+endif::[]
```

Comment:
I don't think you need to include the closing conditional distro, but it helps when multiple conditionals are going on.

```suggestion
endif::openshift-rosa-hcp[]
```

Reply:
ah, good point....thx!